Backend/BikesRentalServer.Tests/StationsService/GetAllBikesAtStation.cs

```diff
@@ -12,11 +14,26 @@ public class GetAllBikesAtStation
     {
         private readonly DatabaseContext _dbContext;
         private readonly Services.StationsService _stationsService;
-        
+        private readonly User _user;
+
         public GetAllBikesAtStation()
         {
             _dbContext = MockedDbFactory.GetContext();
-            _stationsService = new Services.StationsService(_dbContext);
+
+            _user = _dbContext.Users.Add(new User
+            {
+                Username = "test_user",
+                State = UserState.Active,
+                Role = UserRole.User,
+                Reservations = new List<Reservation>(),
+            })

```

Comment:
Wetnij ten nawias o jednego taba, bo tak to `.Entity` dziwnie wisi (to odnośnie każdego miejsca gdzie tak jest)

Reply:
Roger